gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -112,8 +112,14 @@ mv %{buildroot}%{gem_instdir}/bundler.d/%{plugin_name}.rb \
 
 # sample config
 mkdir -p %{buildroot}%{foreman_proxy_settingsd_dir}
-mv %{buildroot}%{gem_instdir}/settings.d/%{plugin_name}.yml.example \
-   %{buildroot}%{foreman_proxy_settingsd_dir}/%{plugin_name}.yml
+<% config_file_paths = spec.files.select { |f| f.start_with?('config/') } +
+                       spec.files.select { |f| f.start_with?('settings.d/') } %>
+<% config_file_paths.map! { |config_file| config_file.chomp('.example') } %>
```

Comment:
This is chomping .example off of the source path as well as the destination, so for example:

<pre>
$ gem2rpm -t smart_proxy_plugin.spec.erb --fetch smart_proxy_remote_execution_ssh | grep yml
mv %{buildroot}%{gem_instdir}/settings.d/remote_execution_ssh.yml \
   %{buildroot}%{foreman_proxy_settingsd_dir}/remote_execution_ssh.yml
mv %{buildroot}%{gem_instdir}/settings.d/smart_proxy_remote_execution_ssh.yml \
   %{buildroot}%{foreman_proxy_settingsd_dir}/smart_proxy_remote_execution_ssh.yml
</pre>

It should be copying `remote_execution_ssh.yml.example` to `remote_execution_ssh.yml`.